src/Razor/test/Microsoft.VisualStudio.LanguageServer.ContainedLanguage.Test/DefaultLSPDocumentSynchronizerTest.cs

```diff
@@ -26,6 +26,7 @@ public DefaultLSPDocumentSynchronizerTest()
 
         private ITextBuffer VirtualDocumentTextBuffer { get; }
 
+#pragma warning disable VSTHRD200 // Use "Async" suffix for async methods
```

Comment:
Unrelated refactorings to avoid warnings.

Reply:
Instead of adding this warning here ignore it at the csproj level (we already have others listed). We don't care about these warnings at all for test files.